tests/check/check_test.go

```diff
@@ -126,3 +132,64 @@ func runTests(t *testing.T, client pb.OpenFGAServiceClient, tests checkTests) {
 		})
 	}
 }
+
+func testBadAuthModelID(t *testing.T, engine string) {
```

Comment:
I had considered adding this test as part of testCheck.  However, those tests are more focused on relationship between model/tuples/check tuples while for this test, there are no relationships between tuple/model/check tuples. Thus, it may make more sense to separate this out.

Reply:
Feel free to unresolve this if you feel we should move this as part of testCheck